src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,94 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int INITIAL_CAPACITY = 10;
```

Comment:
```suggestion
    private static final int INITIAL_CAPACITY = 10;
```
так як нам ця змінна не потрібна за межами класу краще одразу зробити її приватною